recipes-extended/images/apu2-diagnostic-image.bb

```diff
@@ -0,0 +1,15 @@
+require recipes-extended/images/docker-image-minimal.bb
+
+IMAGE_INSTALL_append = " \
+    flashrom \
```

Comment:
@Asiderr I do not think your flashrom recipe is being picked up here. I believe the one from meta-openembedded is being used. You can verify that in your build dir.